autogen/main/cluster.tf.tmpl

```diff
@@ -56,7 +56,7 @@ resource "google_container_cluster" "primary" {
 {% endif %}
 
   subnetwork         = data.google_compute_subnetwork.gke_subnetwork.self_link
-  min_master_version = local.master_version
+  min_master_version = var.release_channel != null ? null : local.master_version
```

Comment:
release_channel is beta so we would need to modify it such that it is not introduced in non beta modules

Reply:
Correct. @kunalkg11 You'll need to make this conditional.

Reply:
Fixed.